server/ngwmn/tests/test_views.py

```diff
@@ -49,7 +60,22 @@ def test_best_case(self, mocker):
         mocker.get(self.stats_overall_url, text=self.mock_overall_json, status_code=200)
         mocker.get(self.stats_monthly_url, text=self.mock_monthly_json, status_code=200)
 
-        response = self.app_client.get(self.site_loc_url)
+        response = self.app_client.get(self.site_loc_url_1)
+        self.assertEqual(response.status_code, 200)
+
+    # Tests if a nested site (a monitoring location with the same geographic coordinates as another monitoring location)
+    # can be selected by site id
+    @requests_mock.Mocker()
+    def test_nested_monitoring_location(self, mocker):
```

Comment:
I don't think this second test really tests if the correct feature is returned (The above test would have worked fine before your change). One way to check that the correct feature is used would be check the contents of the response for <title> tag for the site name. You would want distinct SITE_NAME property for each feature then so would need to update the TEST_SUMMARY_JSON. You could check the same thing on the first test to make sure it's the other SITE_NAME.

Reply:
Okay. I think I made the changes needed to confirm that the correct monitoring location is in the response.